<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamsRollout</class>
 <widget class="QWidget" name="ParamsRollout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Params:</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QListWidget" name="listParams"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>133</height>
      </size>
     </property>
     <property name="title">
      <string>Edit:</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>23</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtParamName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>48</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Begin:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinParamBegin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>44</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>71</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>End:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinParamEnd">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>68</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>95</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Loop:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinParamLoop">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>92</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>44</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemove">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExport">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>68</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Export...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnImport">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>92</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Import...</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
